modules/rubiconBidAdapter.js

```diff
@@ -295,6 +295,20 @@ export const spec = {
             }]
           });
         }
+
+        // support shared id
+        if (bidRequest.userId.sharedid) {
+          data.user.ext.eids.push({
```

Comment:
why not use the functionality in eids.js? https://github.com/prebid/Prebid.js/blob/master/modules/userId/eids.js#L109 Thiis seems to leave an opportunity for these to get out of sync, as they already seem to be for liveramp above.

Reply:
agreed...why not just use `getUserIdsAsEids()` and then filter the ones you don't need server side, this way you avoid client-side updates when you are ready to support other modules?

Reply:
the changes you made, @idettman, still seem overly complicated compared to just getting the eids array from the existing object and passing them through. but it's your adapter, so if you feel that this is flexible enough for your needs and easy enough to change when id modules update their eids, it's your call.

Reply:
@smenzer Our adapter required two things that didn't allow for the `userIdAsEids` array to be copied to the oRTB eids property unmodified. 

1. We need to set additional bidderRequest props for specific eids, so the `callback` property handles those side effects. 
2. Currently only a subset of the id modules are supported by our adapter.